contracts/standard/arbitration/CentralizedCourt.sol

```diff
@@ -0,0 +1,231 @@
+        Dispute storage dispute = disputes[_disputeID];
+        VoteCounter storage voteCounter = dispute.voteCounter;
+        Arbitrable arbitrated = dispute.arbitrated;
+        dispute.arbitrated = Arbitrable(0x0); // Clean up to get gas back and prevent calling it again.
+        dispute.status = DisputeStatus.Solved;
+
+        arbitrated.rule(_disputeID, voteCounter.winningChoice);
+    }
+
+    /** @dev Calculates and tranfers the fee to the court member
+    *   @param _disputeID ID of the dispute to rule 
+    */
+    function withdraw(uint _disputeID) public onlyCourtMember {
+        Dispute storage dispute = disputes[_disputeID];
+        require(dispute.canCollectFee[msg.sender]);
+
+        dispute.canCollectFee[msg.sender] = false;
+        uint feeInWei;
+
+        if(dispute.votes.length == members.length) {
+            feeInWei = dispute.fee/members.length;
+        } else {
+            feeInWei = dispute.fee/dispute.votes.length;
+        }
+        msg.sender.transfer(feeInWei);
+    }
+
+    /** @dev Forces a decision by timeout, fee is splited between court members who voted
+    *   @param _disputeID ID of the dispute to rule
+    */
+    function timeoutDecision(uint _disputeID) public {
+        Dispute storage dispute = disputes[_disputeID];
+        require(now >= dispute.deadline);
+        require(dispute.status == DisputeStatus.Waiting);
+       
+        for(uint8 i = 0; i < dispute.votes.length; i++){
+            dispute.canCollectFee[dispute.votes[i].account] = true;
+        }
+
+        executeRuling(_disputeID);
+    }
+
+    /** @dev Return the status of a dispute.
+     *  @param _disputeID ID of the dispute to rule.
+     *  @return status The status of the dispute.
+     */
+    function disputeStatus(uint _disputeID) public view returns(DisputeStatus status) {
+        return disputes[_disputeID].status;
+    }
+
+    /** @dev Return the ruling of a dispute.
+     *  @param _disputeID ID of the dispute to rule.
+     *  @return ruling The ruling which would or has been given.
+     */
+    function currentRuling(uint _disputeID) public view returns(uint ruling) {
+        Dispute storage dispute = disputes[_disputeID];
+        VoteCounter storage voteCounter = dispute.voteCounter;
+        return voteCounter.winningChoice;
+    }
+
+    /** @dev Cost of appeal. Since it is not possible, it's a high value which can never be paid.
+     *  @param _disputeID ID of the dispute to be appealed. Not used by this contract.
+     *  @param _extraData Not used by this contract.
+     *  @return fee Amount to be paid.
+     */
+    function appealCost(uint _disputeID, bytes _extraData) public view returns(uint fee) {
+        return NOT_PAYABLE_VALUE;
+    }
+
+    /** @dev Set the arbitration price. Only callable by the owner.
+     *  @param _arbitrationPrice Amount to be paid for arbitration.
+     */
+    function setArbitrationPrice(uint _arbitrationPrice) public onlyOwner {
```

Comment:
To update arbitration price, we'd need a majority of court members voting for it, not just the creator.